### PR TITLE
logging: adds Logshuttle logging endpoint support

### PR DESCRIPTION
**Pull request by mccurdyc**
## Relevant (Referenced) Link(s)

* [Fastly API Docs](https://docs.fastly.com/api/logging#api-section-logging_logshuttle)
* [fastly/go-fastly](https://github.com/fastly/go-fastly/blob/c2c2c62210800daab641161412fbe210004aac0a/fastly/logshuttle.go)

## Validation

```
$ make test
```

```
git clone git@github.com:mccurdyc/cli.git /tmp/cli && \
(
    cd /tmp/cli && \
    git checkout mccurdyc/logging-logshuttle && \
    make test && \
    rm -rf /tmp/cli \
)
```

### CODE REVIEW

File: pkg/logging/logshuttle/create.go [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+	common.Base
+	manifest manifest.Data
+	Input    fastly.CreateLogshuttleInput
+}
+
+// NewCreateCommand returns a usable command registered under the parent.
+func NewCreateCommand(parent common.Registerer, globals *config.Data) *CreateCommand {
+	var c CreateCommand
+
+	c.Input = fastly.CreateLogshuttleInput{
```
Comment (mccurdyc) [inline pkg/logging/logshuttle/create.go:25]: Need the comment?

**Comment by pteichman** [comment]
Conflict fix is at https://github.com/fastly/cli/commit/260b2251233f5649e0bd389c720be6dbe5c65e5d

**Comment by mccurdyc** [comment]
Force-pushed Peter's commit.

**Comment by mccurdyc** [comment]
Force-pushed rebasing master.

**Comment by mccurdyc** [comment]
Force-pushed a squashed commit improving test suite.

**Comment by mccurdyc** [comment]
Force-pushed rebasing master.

**Pull request merged**